app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -180,15 +180,18 @@ def actor
 
     ##
     # @return [#errors]
+    # rubocop:disable Metrics/MethodLength
     def create_valkyrie_work
       form = build_form
-      return after_create_error(form_err_msg(form)) unless form.validate(params[hash_key_for_curation_concern])
+      # fallback to an empty array to avoid: # NoMethodError: undefined method `has_key?' for nil:NilClass
+      original_input_params_for_form = params[hash_key_for_curation_concern] ? params[hash_key_for_curation_concern] : {}
```

Comment:
adjusted this error handling so that `generic_works_controller_json_spec.rb:58` passes. this is also how the #create method in this same file handles the params